cfme/tests/openstack/cloud/test_instances.py

```diff
@@ -148,4 +145,22 @@ def test_delete_instance(new_instance):
 
     assert new_instance.name not in new_instance.provider.mgmt.list_vm()
     navigate_to(new_instance, 'AllForProvider')
-    assert new_instance.name not in Quadicon.all()
+    assert new_instance.name not in [q.name for q in Quadicon.all()]
+
+
+def test_list_vms_infra_node(provider, soft_assert):
+    navigate_to(provider.infra_provider, 'ProviderNodes')
+    # Match hypervisors by IP with count of running VMs
+    hvisors = dict()
+    for hv in provider.mgmt.api.hypervisors.list():
+        hvisors.update({hv.host_ip: hv.running_vms})
+
+    # Skip non-compute nodes
+    quads = [q.name for q in Quadicon.all() if 'Compute' in q.name]
+    for quad in quads:
+        host = Host(quad, provider=provider.infra_provider)
+        navigate_to(host, 'Details')
+        host_ip = host.get_detail('Properties', 'IP Address')
+        vms = int(host.get_detail('Relationships', 'VMs'))
```

Comment:
I wonder if we can get the number of VM's from the quad icon, by fetching the details from class flobj 72